src/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react';
+
+export function useLocalStorage(key, initialValue) {
+  const [storedValue, setStoredValue] = useState(() => {
+    try {
+      const item = window.localStorage.getItem(key);
+      return item ? JSON.parse(item) : initialValue;
+    } catch (error) {
+      console.error(error);
+    }
+  });
+
+  const setValue = (value) => {
+    try {
+      setStoredValue(value);
+      debugger;
```

Comment:
Does this `debugger` statement need to be here?

Reply:
LMAO HAHAH certainly not!! :rofl::rofl: